examples/sim_dcmotor_singlephase.py

```diff
@@ -0,0 +1,47 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Example of a DC motor being simulated for a set amount of time. 
+"""
+
+from prog_models.models import dcmotor_singlephase
+
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def run_example():
+
+    motor = dcmotor_singlephase.DCMotorSP()
+    
+    def future_loading(t, x=None):
+        f = 0.5
+        # load proportional to rotor speed
+        if x is None:
+            t_l = 0.0
+        else:
+            t_l = 1e-5 * x['v_rot']**2.0
+        return motor.InputContainer({
+            'v': 10.0 + 2.0 * math.sin(2.0 * math.pi * f * t),
```

Comment:
What is the significance of this equation? Also, you can use tau instead of 2pi
```suggestion
            'v': 10.0 + 2.0 * math.sin(math.tau * f * t),
```